cmd/snap/cmd_info.go

```diff
@@ -99,6 +106,15 @@ func tryDirect(w io.Writer, path string, verbose bool) bool {
 		return false
 	}
 
+	var sha3_384 string
+	if verbose {
+		var err error
+		sha3_384, _, err = asserts.SnapFileSHA3_384(path)
```

Comment:
this will break for directories:
```console
~/.../github.com/snapcore/snapd$ /tmp/snp info ~/src/httpie-snap/
path:    "/home/john/src/httpie-snap/"
name:    http
summary: "HTTPie in a snap"
version: 0.9.6-0 -
~/.../github.com/snapcore/snapd$ /tmp/snp info --verbose ~/src/httpie-snap/
argument: "/home/john/src/httpie-snap/"
warning:  not a valid snap
error: no valid snaps given
~/.../github.com/snapcore/snapd$ 
```